packages/styles/mixins/slider.scss

```diff
@@ -0,0 +1,17 @@
+@mixin thumb {
```

Comment:
Range input pseudo-elements are still pretty different from each other. I still found it useful to organize shared styles, and then add back individual vendor overrides as needed.

Reply:
👍